src/main/java/com/alipay/oceanbase/rpc/mutation/result/BatchOperationResult.java

```diff
@@ -26,13 +27,56 @@ public class BatchOperationResult {
 
     private List<Object> results;
 
+    boolean              hasError = false;
+
     /*
      * construct with List of Object
      */
     public BatchOperationResult(List<Object> results) {
         this.results = results;
     }
 
+    public List<Object> getResults() {
+        return results;
+    }
+
+    public List<Integer> getErrorCodeList() {
+        List<Integer> errorCodeList = new ArrayList<Integer>();
+        for (Object item : results) {
+            int errorCode = ResultCodes.OB_SUCCESS.errorCode;
+            if (item instanceof ObTableException) {
+                errorCode = ((ObTableException) item).getErrorCode();
+                hasError = true;
+            }
+            errorCodeList.add(errorCode);
+        }
+        return errorCodeList;
+    }
+
+    public boolean hasError() {
```

Comment:
Since a batch operation is atomic in one tablet, I guess we don't have a single error in our batch result.  
Could you please verify if this method is applicable?

Reply:
In the old request format, if the batch operation is non-atomic, this method is useful.